src/fabric/src/fabric2_fdb.erl

```diff
@@ -172,7 +174,8 @@ create(#{} = Db0, Options) ->
     % Eventually DbPrefix will be HCA allocated. For now
     % we're just using the DbName so that debugging is easier.
     DbKey = erlfdb_tuple:pack({?ALL_DBS, DbName}, LayerPrefix),
-    DbPrefix = erlfdb_tuple:pack({?DBS, DbName}, LayerPrefix),
+    #{db_prefix_allocator := DBPrefixAllocator} = erlfdb_directory:root(),
```

Comment:
Maybe we'd want an allocator instance under the LayerPrefix and not use the main directory one just to keep things isolated. Without reading the hca code too closely, I assume that's where it would store its counters. We should do a  quick benchmark at least to see if we see considerable contention when making lots of new databases in parallel. I suspect we won't but it's worth checking it out perhaps...

Reply:
`DbPrefixAllocator` is defined in https://github.com/cloudant-labs/couchdb-erlfdb/pull/16/files#diff-cf5acaebd8465ae3276370bbf8d7aaf1R380-R383 with different `DbPrefix`. It will be different from `NodeIdAllocator`.

Reply:
We should probably not mix CouchDB internals with the erlfdb library. At first glance I would expect us not to touch the erlfdb library with this work. `Db` in `erlfdb`-land is an FDB database but here we are dealing with CouchDb `Db` instances so it already gets confusing.

Would this work `erlfdb_hca:create(erlfdb_tuple:pack({?HCA}, LayerPrefix))` to create an allocator instance for our layer prefix only. We are not going for full on FDB directories and nodes just need a short and unique prefix.